android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/PackageActivity.java

```diff
@@ -152,18 +151,18 @@ public void onPageFinished(WebView view, String url) {
       }
     });
 
-    // Determine if ad-hoc distributed KMP contains welcome.htm (case-insensitive) to display
-    FileFilter welcomeFilter = new FileFilter() {
+     // Determine if ad-hoc distributed KMP contains readme.htm (case-insensitive) to display
```

Comment:
This PR is mostly there!

My comment is for the code block in lines 89-92
```
   // Silent installation (skip displaying welcome.htm and user confirmation)
    if (silentInstall) {
      installPackage(context, pkgTarget, pkgId);
      return;
    }
```

1. can you update the comment to include readme.htm and welcome.htm?
2. for `silentInstall`, we don't want to display any of the htm files. Can you update `installPackage()` / `loadWelcomePage()` to handle that?